src/xdocs/beginning_development.xml

```diff
@@ -31,6 +31,14 @@
             git clone git@github.com:you_user_name/checkstyle.git
         </source>
       </div>
+      <p>
+        4. Before opening project in IDEA do build in terminal by running in repository root folder
```

Comment:
`IDEA` => `IDE`.

This is not IDEA specific instructions, so we should make it clear to the user it is for any IDE they are using and not to skip if they are not using IDEA.

Reply:
fixed.